src/components/Application.tsx

```diff
@@ -88,6 +88,10 @@ interface State {
   searchCriteria?: SearchCriteria
   searchError?: any
   searchResults?: beachfront.ImageryCatalogPage
+
+  // Inactivity Timeout state
+  idleTime?: number
```

Comment:
Don't forget to add an initial values for this properties to `generateInitialState()`:

https://github.com/venicegeo/bf-ui/blob/ef360e221a07551c8d7eb152dbd36098e49bdcf2/src/components/Application.tsx#L648-L676